tests/test_indicator.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+test_indicator 
+
+@author: jdiedrichsen
+"""
+
+import unittest 
+import pyrsa.util as rsu
+import numpy as np 
+
+class TestIndicator(unittest.TestCase): 
+    
+    def test_identity(self):
+        a = np.array(range(0,5))
+        a = np.concatenate((a,a))
+        X = rsu.indicator.identity(a)
+        n_row,n_col = X.shape
+        self.assertEqual(n_row,10)
+        self.assertEqual(n_col,5)
+
+    def test_identity_pos(self):
+        a = np.array(range(0,5))
+        a = np.concatenate((a,a))
+        X = rsu.indicator.identity_pos(a)
+        n_row,n_col = X.shape
+        self.assertEqual(n_row,10)
+        self.assertEqual(n_col,4)
+
+    def test_allpairs(self):
+        a = np.array(range(0,5))
+        X = rsu.indicator.allpairs(a)
+        n_row,n_col = X.shape
+        self.assertEqual(n_row,10)
+        self.assertEqual(n_col,5)
```

Comment:
In the future, it may be useful also to test the contents, e.g. by testing one specific cell. Unless only the shapes of these matrixes matter? I am not very familiar with the concept of indicator matrices.

Reply:
Thanks for the note! Will take a look and change in the next few pushes. I think the shape is important to check to avoid dimension conflicts, while adding content check will definitely be essential as well.

Reply:
Fixed and pushed. As suggested, I added a simple content check. For more extensive content checks, I think Joern would be a better person to add them as he first implemented indicator functions and is more familiar to the concept, and might know which cells are more important to add in test cases.